passion.css

```diff
@@ -0,0 +1,53 @@
+body {
+    margin: 0;
+}
+
+.header {
+    display: block;
+    width: 850px;
+    height: 550px;
+    margin-left: auto;
+    margin-right: auto;
+    background-image: url(img/billboard.jpg);
+}
+
+.logo {
+    display: flex;
+}
+
+.logo_center {
+    height: 20%;
```

Comment:
not possible to use normally height with percentage value. remove please this line